src/modules/dashboard/components/Dialogs/ControlSafeDialog/ControlSafeForm.tsx

```diff
@@ -498,14 +518,15 @@ const ControlSafeForm = ({
           onClick={() =>
             showPreview ? handleSubmit() : handleShowPreview(!showPreview)
           }
-          text={showPreview ? MSG.buttonConfirm : MSG.buttonCreateTransaction}
```

Comment:
We could use the existing pattern `{id: button.confirm}` and `{id: button.createMotion}` here?

Reply:
![9780312586393](https://user-images.githubusercontent.com/18473896/197063393-0b2f5928-0f9a-47e7-ba21-a009b1dba73c.jpg)